pints/_error_measures.py

```diff
@@ -272,6 +272,35 @@ def __call__(self, x):
             (self._problem.evaluate(x) - self._values)**2))
 
 
+class NormalisedRootMeanSquaredError(ProblemErrorMeasure):
+    """
+    Calculates a normalised root mean squared error (RMSE):
```

Comment:
This makes me think (normalised root mean square error) = (RMSE)
Maybe clearer to write "normalised RMSE (root mean square error)" in this case?

You might even want to break it up into multiple sentences, e.g.

```
"""
Calculates a normalised root mean square error.

This is similar to the (unnormalised) root mean square error (RMSE) defined as
``...``
but
...
```
?

Reply:
Also would it be clearer to define it as ``sqrt(....) / N`` ? (even though in code it's better the way you have it)

Reply:
Done!